scripts/lint-unicode.py

```
@@ -0,0 +1,114 @@
+                pass
+    return frozenset(cps)
+
+
+def is_suspicious(ch, allow):
+    cp = ord(ch)
+    if cp in ALLOWED or cp in allow:
+        return False
+    return unicodedata.category(ch) in ('Cf', 'Cc')
+
+
+def main():
```

Ruff (0.15.20)

[warning] Too many branches (13 > 12)

(PLR0912)